integration_tests/rtm/test_rtm_client.py

```diff
@@ -0,0 +1,201 @@
+import asyncio
+import collections
+import logging
+import os
+import threading
+import time
+import unittest
+
+import psutil
+import pytest
+
+from integration_tests.env_variable_names import SLACK_SDK_TEST_CLASSIC_APP_BOT_TOKEN, \
+    SLACK_SDK_TEST_RTM_TEST_CHANNEL_ID
+from integration_tests.helpers import async_test
+from slack import RTMClient, WebClient
+
+
+class TestRTMClient(unittest.TestCase):
+    """Runs integration tests with real Slack API"""
+
+    def setUp(self):
+        if not hasattr(self, "logger"):
+            self.logger = logging.getLogger(__name__)
+            self.channel_id = os.environ[SLACK_SDK_TEST_RTM_TEST_CHANNEL_ID]
+            self.bot_token = os.environ[SLACK_SDK_TEST_CLASSIC_APP_BOT_TOKEN]
+
+        if not hasattr(self, "cpu_monitor") or not TestRTMClient.cpu_monitor.is_alive():
+            def run_cpu_monitor(self):
+                self.logger.debug("Starting CPU monitor in another thread...")
+                TestRTMClient.cpu_usage = 0
+                while True:
+                    p = psutil.Process(os.getpid())
+                    current_cpu_usage: float = p.cpu_percent(interval=0.5)
+                    self.logger.debug(current_cpu_usage)
+                    if current_cpu_usage > TestRTMClient.cpu_usage:
+                        TestRTMClient.cpu_usage = current_cpu_usage
+
+            TestRTMClient.cpu_monitor = threading.Thread(target=run_cpu_monitor, args=[self])
```

Comment:
Although the CPU usage of running this thread in the same process is ignorable, it may be also fine to have another process instead.